src/signTx.c

```diff
@@ -1604,6 +2041,14 @@ static void signTx_handleConfirm_ui_runStep()
 
 	UI_STEP_BEGIN(ctx->ui_step, this_fn);
 
+	UI_STEP(HANDLE_CONFIRM_STEP_TXID) {
+		ASSERT(ctx->commonTxData.txSigningMode == SIGN_TX_SIGNINGMODE_PLUTUS_TX);
+		ui_displayHexBufferScreen(
+		        "Tx body hash",
```

Comment:
Why did you change this from `Transaction id` to `Tx body hash`? I thought transaction id is the term used by public.

Reply:
At one point, I was not sure whether tx id really is equal to tx body hash, and made this change to remind me of it. Apparently it got committed into an inappropriate branch.